script/heroku-postbuild.sh

```diff
@@ -0,0 +1,14 @@
+# define absolute paths to each repo
+vagov_apps_dir=${PWD}
+vagov_content_dir=${vagov_apps_dir}/../vagov_content
+
+echo Installing vagov_content into ${vagov_content_dir}
+
+# clone the content into the sibling directory
+git clone --depth=1 https://github.com/department-of-veterans-affairs/vagov-content ${vagov_content_dir}
+
+# cd back to the apps
+cd vagov_apps_dir
+
+# build the site as usual
+npm run build --  --entry static-pages,style
```

Comment:
This is how review instances will continue to be created on the `vets-website` side of things.

Reply:
I think because those entries are global entry points, they're always compiled. Here's the output from a review instance at `vagov-content that uses the same build command -

```
Hash: d41cc4c610fc620662a2
       Version: webpack 4.1.1
       Time: 23520ms
       Built at: 2018-10-24 13:24:28
       Entrypoint static-pages [big] = vendor.entry.js vendor.css static-pages.entry.js static-pages.css
       Entrypoint styleConsolidated [big] = styleConsolidated.entry.js styleConsolidated.css
       Entrypoint style [big] = style.entry.js style.css
       Entrypoint polyfills [big] = vendor.entry.js vendor.css polyfills.entry.js
       Entrypoint brandConsolidation [big] = brandConsolidation.entry.js brandConsolidation.css
       Build finished!
```

Reply:
Ah, yep, I forgot that was how it worked. I think you can remove style, then.